### ISSUE TITLE
Stuck migration on tool 

### ISSUE THREAD

**Issue by theLargePhoenix** [JannikBirn/palworld-host-save-fix#19]
Started the migration process but its been more than 20 minutes of waiting im assuming its because of "C:/Program Files (x86)/Steam/steamapps/common/PalServer/Pal/Saved/SaveGames/0/E86F2D53432C72BF65922AA6BEF545D3\Players\5368B079000000000000000000000000.sav.json already exists, this will overwrite the file
Are you sure you want to continue? (y/n): " on the automated script windwo? im not sure if i should just type y

**Comment by BlameBlight** [comment]
Yes, the latest update to palworld broke this. Upon telling it to perform the overwrite which it was not doing previously it now does this > You have old user that you want which we will call user 1, User 2 is the brand new character that we are trying to replace with user 1. Upon telling to do the overwrite it gets confused and tries to overwrite user 2 with user 2 again and fails. Instead of taking user one and merging it with 2 as expected
Now its just a waiting game as hopefully Jannik can fix this.

**Comment by JannikBirn** [comment]
> You have old user that you want which we will call user 1, User 2 is the brand new character that we are trying to replace with user 1. Upon telling to do the overwrite it gets confused and tries to overwrite user 2 with user 2 again and fails. Instead of taking user one and merging it with 2 as expected

I don't really understand how this is the case for you. How did you tell that this is what happened?
